NitroxServer/Communication/Packets/Processors/StorageSlotRemoveItemProcessor.cs

```diff
@@ -0,0 +1,28 @@
+﻿using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic;
+using NitroxServer.GameLogic.Items;
+
+namespace NitroxServer.Communication.Packets.Processors
+{
+    class StorageSlotRemoveItemProcessor : AuthenticatedPacketProcessor<StorageSlotItemRemove>
+    {
+        private readonly PlayerManager playerManager;
+        private readonly InventoryData inventoryData;
+
+        public StorageSlotRemoveItemProcessor(PlayerManager playerManager, InventoryData inventoryData)
+        {
+            this.playerManager = playerManager;
+            this.inventoryData = inventoryData;
+        }
+
+        public override void Process(StorageSlotItemRemove packet, Player player)
+        {
+            // Only need to send this, if it is a shared owner
```

Comment:
What do you mean by shared owner?

Reply:
Bad wording. It means if someone tries to remove a battery, that is not there in the first place, it will not send the info to other players (cause slot is empty anyways)